RunForrest/src/js/main.js

```diff
@@ -0,0 +1,30 @@
+function validate(e) {
+    if (e.keyCode===13) callPromise();
+ }
+
+ function callPromise(){
+    fetch("https://api.myjson.com/bins/uptto")
+    .then(Response => Response.json())
+    .then(function(json) {
```

Comment:
Use arrow functions, like line 7.
```
.then((json) => {
 // ...
}
```

https://medium.freecodecamp.org/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26